### PR TITLE
Update for 2024.

### PR DESCRIPTION
**Pull request by jonathanbull**

### CODE REVIEW

File: country/australia.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ You cannot use harvested or scraped lists in Australia. Under the spam legislati
 Some organisations are exempt from the legislation. These include registered charities, educational institutions (only when sent to current or former students), government bodies and registered political parties. These messages must relate to goods or services offered by the exempt organisation.
 
 ## Penalties
-According to the ACMA, “the penalty units referred to in the Spam Act are currently equal to $180 each. For example, the penalty under section 25(5)(b) of the Spam Act for a company with a previous record of spamming and who sent two or more spam messages on a given day without consent is a maximum fine of 10,000 penalty units. This equates to a maximum penalty of $1,800,000 per day.”
+According to the ACMA, "the penalty units referred to in the Spam Act are currently equal to $180 each. For example, the penalty under section 25(5)(b) of the Spam Act for a company with a previous record of spamming and who sent two or more spam messages on a given day without consent is a maximum fine of 10,000 penalty units. This equates to a maximum penalty of 1,800,000 AUD per day.
```
Comment (thoeva) [inline country/australia.md]: This was a direct quote from the ACMA website. Did you mean to remove the quotation marks, and if it is a quotation mark should we be changing the currency format? 

In any event we should be consistent here.

File: country/china.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # China
-China's email marketing regulations are set out in the Regulations of Email Services (RES). The regulations are far more restrictive than the USA's CAN-SPAM Act.
+China's email marketing regulations are set out in the Regulations of Email Services (RES). These regulations are far stricter than those seen in other countries.
```
Comment (thoeva) [inline country/china.md:2]: Far stricter than which countries? It doesn't appear to immediately be more strict than the GDPR, but is more than the Indian and USA legislation. Should we be specific here or just remove it – ensuring it's factual.

File: country/finland.md [hunk 3]
```diff
@@ -1,28 +1,22 @@
...
 Email is also allowed if the recipient has not specifically opted out and if you obtain the customer's contact details in the context of sale of a product or service. You cannot sell the data, however you can use the contact information for direct marketing of your own similar products. This is essentially soft opt-in using implied consent.
 
 ## Notable exceptions or requirements
-
 As Finland follows the GDPR, it requires that your company must have a data protection officer. The person is in charge of maintaining and enforcing data security standards.
 
 ## Penalties
-
-The maximum fine available under the GDPR is up to €20 million, or 4% annual global turnover – whichever is higher. "The Office of the Data Protection Ombudsman" will be the body that will safeguard rights and freedoms when processing personal data.
+The maximum fine available under the GDPR is up to 20 million EUR, or 4% annual global turnover – whichever is higher. "The Office of the Data Protection Ombudsman" is the body that will safeguard rights and freedoms when processing personal data.
```
Comment (thoeva) [inline country/finland.md]: > "The Office of the Data Protection Ombudsman"
We're a bit inconsistent in where we use quotation marks when referring to the authority. They were removed in some (i.e Denmark) but not here.

File: country/germany.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # Germany
-Germany has some of the strictness email marketing laws in the world. The German regulations are defined by the Federal Data Protection Act, the GDPR, and the Telemedia Act.
```
Comment (thoeva) [inline country/germany.md:2]: Glad we fixed this!

File: country/ireland.md [hunk 5]
```diff
@@ -1,27 +1,21 @@
...
-
 As Ireland follows GDPR, it requires that companies have a data security officer. The person is in charge of maintaining and enforcing data security standards.
 
 ## Penalties
+The maximum fine available under the GDPR is up to 20 million EUR, or 4% annual global turnover – whichever is higher. The "Data Protection Commissioner" is the supervisory authority.
 
-The maximum fine available under the GDPR is up to €20 million, or 4% annual global turnover – whichever is higher. The "Data Protection Commissioner" will be  the supervisory authority here.
-
-In addition to a fine under GDPR, the  Irish government will also impose a fine up to EUR 250,000 per message sent by a company and an individual may be fined up to EUR 50,000 per message
+In addition to a fine under GDPR, the Irish government will also impose a fine up to 250,000 EUR per message sent by a company and an individual may be fined up to 50,000 EUR per message
```
Comment (thoeva) [inline country/ireland.md]: > Irish government will also impose a

I reckon this should be "can".

File: country/israel.md [hunk 6]
```diff
@@ -1,21 +1,16 @@
...
 ## Consent
-
 You can only send marketing emails to those who have provided explicit consent to receive marketing emails from you.
 
 The recipient can also be sent marketing emails if they shared their contact info, when purchasing a service or product, or when negotiating such purchase (specified for general advertising, which includes marketing emails)
 
 ## Penalties
-
-The fine that would be imposed would depend on the severity of breach of the law and could go up to ILS 202,000 in a criminal court. Recipients of spam email may also seek damages which in a civil court could be as much as ILS 1,000 for each message sent by the advertiser to the recipient.
+The fine that would be imposed would depend on the severity of breach of the law and could go up to 202,000 NIS in a criminal court. Recipients of spam email may also seek damages which in a civil court could be as much as ILS 1,000 for each message sent by the advertiser to the recipient.
```
Comment (thoeva) [inline country/israel.md]: > could go up to 202,000 NIS in a 

It's new Israeli shekel (NIS) but the ISO is ILS. Maybe worth checking we're consistent in use as they are interchangeable.

File: country/uk.md [hunk 7]
```diff
@@ -15,9 +15,7 @@ Unsubscribe requests must be processed within 28 days.
 The soft opt-in rule means you may be able to email your own customers, but it does not apply to prospective customers or new contacts (e.g. from bought-in lists). It also does not apply to non-commercial promotions (e.g. charity fundraising or political campaigning).
 
 ## Penalties
-There are large maximum fines that can be levied as penalties in the event of a breach, however the Information Commissioner's Office (ICO) has repeatedly said that it would not make early examples of organisations for minor infringements of the GDPR, nor would maximum fines become the norm.
-
-The maximum fine available under the GDPR is up to €20 million, or 4% annual global turnover – whichever is higher.
+The maximum fine available under the GDPR is up to 20 million EUR, or 4% annual global turnover – whichever is higher.
```
Comment (thoeva) [inline country/uk.md]: > 20 million EUR
17.5 million GBP

File: README.md [hunk 8]
```diff
@@ -1,34 +1,32 @@
...
+| [Japan](/country/japan.md) | ASCT, Anti-Spam Act | Name, mailing address | Yes | Implied consent if you have a previous business relationship, otherwise explicit consent required | Up to 1 million JPY or 1 year of imprisonment |
+| [Singapore](/country/singapore.md) | PDPA, Spam Control Act 2007 | Name, email address | Yes | Explicit consent, via a minimum of soft opt-in | 25 SGD per email, up to 1 million SGD |
+| [South Africa](/country/south-africa.md) | ECTA, CPA, PPIA | Name, email address | Yes | Minimum of implied consent | Fines (no limit) or up to 12 months imprisonment |
+| [United Arab Emirates](/country/uae.md) | RUEC | Name, mailing address | Yes | Implied consent | Fines of up to 10 million AED |
+| [United Kingdom](/country/uk.md) | UK GDPR, PECR, DPA 2018 | Name, mailing address| Yes| Explicit consent, via a minimum of soft opt-in | Up to 20 million EUR, or 4% annual global turnover – whichever is higher |
+| [USA](/country/usa.md) | CAN-SPAM | Name, mailing address, contact information| Yes| Prior consent is not required | Up to $51,744 per violation |
```
Comment (thoeva) [inline README.md]: > Up to $51,744 per violation |

USD?

**Review by thoeva — CHANGES_REQUESTED** [review]
Few comments mainly around consistency of ISO/currency symbols. 

And inconsistency around the use of quotation marks on legislation names

File: country/australia.md (hunk 1 above)
Comment (jonathanbull) [reply country/australia.md]: I can no longer find that quote on the ACMA website and that figure is now outdated. So I have rewritten in https://github.com/threeheartsdigital/email-marketing-regulations/pull/29/commits/3f357095add336b13b78085882413392ace48825

File: country/ireland.md (hunk 5 above)
Comment (jonathanbull) [reply country/ireland.md]: https://github.com/threeheartsdigital/email-marketing-regulations/pull/29/commits/78a98b318bac8854628b7b8926245bd8dea6a535

File: country/israel.md (hunk 6 above)
Comment (jonathanbull) [reply country/israel.md]: Good spot. [gov.il](https://www.gov.il/en/pages/17052018_7) references ILS so I've gone with that. 

https://github.com/threeheartsdigital/email-marketing-regulations/pull/29/commits/2d3e1198005a5df49c80725c34536ea9fa70b684

File: country/uk.md (hunk 7 above)
Comment (jonathanbull) [reply country/uk.md]: https://github.com/threeheartsdigital/email-marketing-regulations/pull/29/commits/d760d0f31f01f375f89124f1f61216f0d21df4a6

File: README.md (hunk 8 above)
Comment (jonathanbull) [reply README.md]: https://github.com/threeheartsdigital/email-marketing-regulations/pull/29/commits/f7138179ccead4c1fcf2d5d0e0ed593ed569e1d3

File: country/finland.md (hunk 3 above)
Comment (jonathanbull) [reply country/finland.md]: Good point. I think I opted for removing quotes as the whole guide is littered with references, the majority of which aren't quoted. I missed this one and a few others. Corrected in https://github.com/threeheartsdigital/email-marketing-regulations/pull/29/commits/2164ae6b39611e3c77293175add2c8dfc3cdbb3f

**Review by thoeva — APPROVED** [review]
All my points from Feb are addressed. Not going through everything else again, as I did that at the time :)

**Pull request merged**